doc/dox_comments/header_files/bn.h

```diff
@@ -26,3 +26,1230 @@
 */
 int wolfSSL_BN_mod_exp(WOLFSSL_BIGNUM *r, const WOLFSSL_BIGNUM *a,
         const WOLFSSL_BIGNUM *p, const WOLFSSL_BIGNUM *m, WOLFSSL_BN_CTX *ctx);
+
```

Comment:
Please revert the changes to this file. We are not exposing openssl compatibility layer documentation. Users should refer to openssl documentation for these API's.